### ISSUE TITLE
greptests but for test helper files

### ISSUE THREAD

**Issue by lydia-duncan** [chapel-lang/chapel#8175]
It'd be helpful to have a way to search through the helper files for our tests.  I've often wanted a way to search through the compopts/execopts files, or the skipifs specifically, to remember how to write a particular operation.

For example:
I recently wanted to know how to write a skipif file that would avoid/include cygwin.  Rather than searching the entire test directory (including the .chpl/.good files), or writing my own script to only look through the .skipif files, it would be nice to have that script readily available.

**Comment by ronawho** [comment]
I'm not opposed to such a tool, but I personally find it easy enough to use use find+grep (or whatever your favorite tool is (git grep, ack, ag, rg, etc.)


```sh
find test/ -iname '*skipif' | xargs grep -i cygwin
```

```sh
git grep cygwin test/*skipif
```


p.s. I _think_ (though I'm not positive) that the tools label is for user facing tools like chpldoc/chplvis/c2chapel, and that BTR might be a better label?

**Comment by lydia-duncan** [comment]
Thanks for the suggestions, I've updated the labels.

I think that line is good for now (and can now point someone to this issue rather than telling them to look up how to write the command), but would still like to have this tool present so scripting knowledge is not a barrier to contributing, so I will leave this issue open.

**Comment by ronawho** [comment]
Right, right -- just to be clear, I meant "I'm fine with such a tool, and here's a 'workaround' in the meantime", not "why would you want such a tool?"

**Comment by Maxrimus** [comment]
Resolved with #17546
